packaging/installer/netdata-updater.sh

```diff
@@ -134,8 +133,7 @@ update() {
     pids=$(pidof netdata)
     do_not_start=
     if [ -n "${pids}" ]; then
-      #shellcheck disable=SC2086
-      kill -USR1 ${pids}
+      kill -USR1 "${pids}"
```

Comment:
This needs to be expanded. `pids` is a space separated list of PID's, which need to be passed as multiple arguments to `kill`.

Reply:
Good point. Fixed.